### ISSUE TITLE
Autocompleted 'It' in AboutStrings.Koans.ps1

### ISSUE THREAD

**Issue by EVWorth** [vexx32/PSKoans#213]
# Describe "Koan Bug, Issue, or Help Request"

After Finishing Koan 107 (Comparison), I ran Measure-Karma and already showed 1/18 finished for AboutStrings.

![image](https://user-images.githubusercontent.com/50814635/60465360-87055d00-9c16-11e9-8afa-f909b7057589.png)

## Context "The Problematic Assertions"

I ran .\AboutStrings.Koans.ps1 and saw the following:

![image](https://user-images.githubusercontent.com/50814635/60465745-7a353900-9c17-11e9-9be3-3cf9effdd99a.png)

Looking at the file, it looks like the Koan is missing its request.
![image](https://user-images.githubusercontent.com/50814635/60465863-b8caf380-9c17-11e9-86ee-2b852a1ab9c1.png)

' ' seems to be the correct answer, but it looks like the Koan should have '__' so the user can remove it.

**Comment by vexx32** [comment]
I agree, this shouldn't complete automatically. I think just replacing one of the `$String` values with a blank is a good way to go. The literal space can stay as is, maybe?

**Comment by vexx32** [comment]
Thanks for closing these, I quite forgot!

@EVWorth for future reference when you do a PR, you can actually have related issues closed automatically when a PR is merged if you mention something along the lines of `Resolves #ISSUENUMBER` in the PR summary. That lets Github know to automatically close the issue when the PR is merged in.

There's a [short list of keywords](https://help.github.com/en/articles/closing-issues-using-keywords) Github recognises for this, so you can use what you prefer. 🙂

**Comment by EVWorth** [comment]
No problem!

I thought there was something like that.

![image](https://user-images.githubusercontent.com/50814635/60593624-c9d74a00-9d68-11e9-8a40-1f788c35d7ac.png)
Looks like thats what I get for being too wordy!

**Comment by vexx32** [comment]
Hahaha so close.... 😄
